tests/functional/phpinfo-1.phpt

```diff
@@ -16,11 +16,17 @@ phpinfo();
 %a
 mongodb
 
-mongodb support => enabled
-mongodb version => 1.%d.%d%S
-mongodb stability => %s
-libmongoc version => 1.%s
-libbson version => 1.%s
+MongoDB support => enabled
+MongoDB extension version => 1.%d.%d%S
+MongoDB extension stability => %s
+libbson bundled version => 1.%d.%d%S
+libmongoc bundled version => 1.%d.%d%S
```

Comment:
libbson and libmongoc follow the same version format with optional trailing strings, so I updated these patterns to match our own.